build/repo.targets

```diff
@@ -0,0 +1,10 @@
+<Project>
+  <Target Name="CleanNpm">
```

Comment:
Do we have a repo-level clean target? It might be good to have one and wire this in to that. If not, we can add that in later.